content/docs/csidriver/features/powerflex.md

```diff
@@ -925,6 +937,25 @@ externalAccess: "10.0.0.0/24"
 
 This means that we allow for NFS Export created by driver to be consumed by address range `10.0.0.0-10.0.0.255`.
 
+## Configuring NFS independent of SDC
+
+Starting from CSM 1.12.0, the CSI PowerFlex driver supports configuring NFS independent of SDC. This separation is helpful in scenarios where an SDC is not available in the cluster or additional network interfaces do not need to be deployed.
+
+To enable this feature you need to disable the automatic `sdc` deployment from the values file and provide the interface names mapping for each of the nodes that are being used.
+
+```
+node:
+  ...
+  sdc:
+    # enabled: Enable/Disable SDC
+    enabled: false
+  ...
+
+interfaceNames:
+  # worker1: interface1
```

Comment:
It might be nice to provide a valid example here, as most times worker1 is not valid and would be something like "worker-1-jxsjoueeewabc.domain". The whole string is needed. Also, interface1 should be the interface id pointing to the IP of the worker itself. Most cases it is ens192 or something like that, but it needs to be translated to the IP of the worker.